tests/fsharp/tests.fs

```diff
@@ -1892,26 +1891,25 @@ module RegressionTests =
         peverify cfg "petshop.dll"
 #endif
 
-    [<Test >]
+    [<Fact>]
     let ``86`` () = singleTestBuildAndRunVersion "regression/86" FSC_OPTIMIZED "supports-ml"
 
-    [<Test >]
+    [<Fact>]
     let ``struct-tuple-bug-1-FSI`` () = singleTestBuildAndRun "regression/struct-tuple-bug-1" FSI
 
 #if !NETCOREAPP
     // This test is disabled in coreclr builds dependent on fixing : https://github.com/dotnet/fsharp/issues/2600
-    [<Test>]
+    [<Fact>]
     let ``struct-measure-bug-1`` () =
         let cfg = testConfig "regression/struct-measure-bug-1"
 
         fsc cfg "%s --optimize- -o:test.exe -g" cfg.fsc_flags ["test.fs"]
 
         peverify cfg "test.exe"
 
-[<NonParallelizable>]
```

Comment:
Do I get it right that the tests were not parallelized anyway (via assembly attribute), so this was a dupe of this idea and can be removed therefore?

Reply:
Yes, it looks there was some fight here and eventually it was deparallelized as a whole assembly.